src/pages/subreddit/Subreddit.js

```diff
@@ -0,0 +1,61 @@
+import React, {useEffect, useState} from 'react';
+import {Link, useParams} from "react-router-dom";
+import './Subreddit.css';
+import axios from "axios";
+
+function Subreddit() {
+    const [data, setData] = useState([]);
```

Comment:
`data` wordt uiteindelijk een object. Een default waarde van `undefined` past hier beter bij
```suggestion
    const [data, setData] = useState();
```